index.js

```diff
@@ -9,15 +9,68 @@ app.set('view engine', 'hbs');
 const menu = {
   1: {
     id: 1,
-    name: "Strawberry cheesecake",
+    name: 'Strawberry cheesecake',
     price: 6
+  },
+
+  2: {
+    id: 2,
+    name: 'deep fried rubberduck',
+    price: 10
+  },
+
+  3: {
+    id: 3,
+    name: ' frozen timer',
+    price: 20
+  },
+
+  4: {
+    id: 4,
+    name: ' pickled rubberduck',
+    price: 30
   }
 };
 
-app.get('/', function(req, res){
+let id = 5;
+
+const orders = {};
+
+let orderid = 1;
+
+app.get('/', function(req, res) {
   res.render('index');
 });
 
-app.listen(8080, function(){
+app.get('/menu', function(req, res) {
+  res.json(menu);
+  // console.log("fetched");
+});
+
+app.post('/api/order', function(req, res) {
+  const index = orderid;
+  orders[index] = req.body;
+  orders[index]['id'] = index;
+  orderid++;
+  res.status(200).json({ OK: 'order completed' });
+});
+
+app.get('/api/order', function(req, res) {
```

Comment:
It could be better to call the route `/api/orders` as it returns multiple orders rather than a single order